src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -0,0 +1,474 @@
+                  v-if="i != 1"
+                  :key="`option-item-${i}`"
+                  :value="i"
+                >
+                  {{ i }}
+                </option>
+              </template>
+            </select>
+
+            <i18n
+              path="modals.createMultisigAccount.consensusRequiredContent"
+              tag="div"
+              class="text"
+            >
+              <span>
+                {{ minNumOfSigners }}
+              </span>
+            </i18n>
+
+            <BtnHelp
+              :title="$t('modals.createMultisigAccount.consensusRequiredHelpTitle')"
+              :msg="$t('modals.createMultisigAccount.consensusRequiredHelpMsg')"
+            />
+          </div>
+        </div>
+      </div>
+    </template>
+    <template #footer>
+      <BtnMain
+        v-if="isCreatingVault || !!multisigAccount"
+        inline
+        nowrap
+        :disabled="isCreatingVault"
+        @click="navigateToMultisigVault"
+      >
+        {{ $t('modals.creatingMultisigAccount.btnText') }}
+      </BtnMain>
+      <BtnMain
+        v-else
+        inline
+        nowrap
+        :disabled="!canCreateMultisig"
+        @click="createMultisigAccount"
+      >
+        {{ $t('modals.createMultisigAccount.btnText') }}
+      </BtnMain>
+    </template>
+  </Modal>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  ref,
+} from '@vue/composition-api';
+import {
+  MODAL_READ_QR_CODE,
+  MODAL_DEFAULT,
+  validateHash,
+  MULTISIG_VAULT_MIN_NUM_OF_SIGNER,
+} from '../../utils';
+import { ICreateMultisigAccount, MultisigAccount, MultisigCreationStep } from '../../../types';
+import { ROUTE_ACCOUNT_DETAILS } from '../../router/routeNames';
+
+import { useMultisig } from '../../../composables';
+import Modal from '../Modal.vue';
+import BtnMain from '../buttons/BtnMain.vue';
+import BtnText from '../buttons/BtnText.vue';
+import BtnHelp from '../buttons/BtnHelp.vue';
+import AvatarWithChainName from '../AvatarWithChainName.vue';
+import InputField from '../InputField.vue';
+import QrScanIcon from '../../../icons/qr-scan.svg?vue-component';
+import PlusCircle from '../../../icons/plus-circle-fill.svg?vue-component';
+import CheckSuccessCircleIcon from '../../../icons/check-success-circle.svg?vue-component';
+
+export default defineComponent({
+  name: 'MultisigVaultCreate',
+  components: {
+    Modal,
+    BtnMain,
+    BtnText,
+    BtnHelp,
+    InputField,
+    AvatarWithChainName,
+    QrScanIcon,
+    CheckSuccessCircleIcon,
+  },
+  props: {
+    resolve: { type: Function, required: true },
+    reject: { type: Function, required: true },
+  },
+  setup(props, { root }) {
+    const { deployMultisigAccount, multisigProgress } = useMultisig({ store: root.$store });
+    const multisigAccount = ref<MultisigAccount | null>(null);
+    const signers = ref<ICreateMultisigAccount[]>([
+      {
+        address: '',
+      },
+      {
+        address: '',
+      },
+    ]);
```

Comment:
```suggestion
    const buildAddressForm = () => ({ address: '' })
 
    const signers = ref<ICreateMultisigAccount[]>([
      buildAddressForm(), buildAddressForm()
    ]);
```

Reply:
To be honest I don't see much of a value from this change.